helper/src/main/java/io/github/libxposed/helpers/MethodFinder.java

```diff
@@ -0,0 +1,199 @@
+package io.github.libxposed.helpers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+/**
+ * Helper for quick and elegant to find method(s).
+ */
+@SuppressWarnings("unused")
+public final class MethodFinder extends BaseFinder<Method, MethodFinder> {
+
+    private MethodFinder(@NonNull Stream<Method> stream) {
+        super(stream);
+    }
+
+    /**
+     * Create MethodFinder with the class.
+     *
+     * @param clazz                   class
+     * @param findSuperClassPredicate find super class predicate(return true = break, false = continue), null if don't find methods in superclass
+     * @return MethodFinder
+     */
+    public static MethodFinder from(@NonNull Class<?> clazz, @Nullable Predicate<Class<?>> findSuperClassPredicate) {
+        Stream<Method> stream = Stream.empty();
+
+        do {
+            stream = Stream.concat(stream, Arrays.stream(clazz.getDeclaredMethods()));
+            stream = Stream.concat(stream, Arrays.stream(clazz.getInterfaces())
+                    .flatMap(c -> Arrays.stream(c.getDeclaredMethods()).filter(Method::isDefault)));
```

Comment:
Inconsistency. For classes, you don't filter abstract methods, but for interfaces, you do.